web/src/components/admin/users/buttons/InviteUserButton.tsx

```diff
@@ -35,17 +38,23 @@ export const InviteUserButton = ({
     {
       onSuccess: () => {
         setShowInviteModal(false);
-        mutate();
+        if (typeof mutate === "function") {
+          mutate();
+        } else {
+          mutate.forEach((fn) => fn());
+        }
```

Comment:
invited users and accepted have different mutate functions, but needed to call both at once to update slack users invite status properly, so added the ability to pass in an array of mutate functions.